app/models/profile.rb

```diff
@@ -13,13 +13,12 @@ def contact(other_profile)
 
     if !other_profile.driver?
       msg = %[
-        RideWithBernie: #{first_name} at #{phone} offered to drive you to #{event_title}, 
-        coming from #{location} with #{seats} seat#{'s' if seats != 1} available.
+        RideWithBernie: #{first_name} at #{phone} from #{location} offered to drive you to #{event_title}. Can you go?
```

Comment:
I think we're gonna get a lot of people replying to this text message directly if we end with a question! I'm gonna make some quick changes.